email/created_reservation.html

```diff
@@ -0,0 +1,46 @@
+<html>
+<body>
+Geachte heer/mevrouw,<br />
+<br />
+Uw reservering is aangemaakt met de volgende gegevens:<br />
+<br />
+<table>
+    <tr>
+        <td style="padding-right: 5px;"><b>Op naam van</b></td>
+        <td>{{inNameOf}}</td>
+    </tr>
+    <tr>
+        <td style="padding-right: 5px;"><b>Email</b></td>
+        <td>{{email}}</td>
+    </tr>
+    <tr>
+        <td style="padding-right: 5px;"><b>Telefoon</b></td>
+        <td>{{telephone}}</td>
+    </tr>
+    <tr>
+        <td style="padding-right: 5px;"><b>Datum/tijd</b></td>
+        <td>{{date}}</td>
+    </tr>
+    <tr>
+        <td style="padding-right: 5px;"><b>Hoeveelheid personen</b></td>
+        <td>{{amountPersons}}</td>
+    </tr>
+    <tr>
+        <td style="padding-right: 5px;"><b>Bijzonderheden</b></td>
+        <td>{{notes}}</td>
+    </tr>
+</table>
+<br />        
+<br />        
+Als uw reservering groter is dan 12 personen zal er contact met u opgenomen worden. Als u vragen heeft of de reservering wilt stoppen kunt u contact opnemen via de telefoon.
```

Comment:
persoonlijk vind ik het netter als er staat: "of de reservering wilt annuleren" ipv "of de reservering wilt stoppen"